orders-service/README.md

```diff
@@ -0,0 +1,142 @@
+- [Kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/) 1.16 or higher.
+- [Helm](https://helm.sh/) 3.0 or higher - not required.
+- [Docker Compose](https://docs.docker.com/compose/) 3.0 or higher - not required.
+
+## Installation
+
+### By Kubectl
+
+To install Orders Service in Kyma cluster, run:
+
+```bash
+kubectl create ns orders-service
+kubectl apply -f ./deployment/orders-service.yaml
+```
+
+To install Serverless Function in Kyma cluster, run:
+
+```bash
+kubectl create ns orders-service
+kubectl apply -f ./deployment/function.yaml
+```
+
+### By Helm
+
+To install Orders Service in Kyma cluster, run:
+
+```bash
+helm install orders-service --namespace orders-service --create-namespace --timeout 60s --wait ./chart
+```
+
+Configuration for helm release is in [`values.yaml`](./chart/values.yaml) file.
+
+### By Docker Compose
+
+To build and run the Orders Service locally with Docker Compose, run:
+
+```bash
+make docker-compose
+```
+
+## Cleanup
+
+### By Kubectl
+
+Run the following command to completely remove the example (Orders Service or Serverless Function) and all its resources from the cluster:
+
+```bash
+kubectl delete all -l app=orders-service -n orders-service
+kubectl delete ns orders-service
+```
+
+### By Helm
+
+Run the following command to completely remove the helm release with example and all its resources from the cluster:
+
+```bash
+helm delete orders-service -n orders-service
+kubectl delete ns orders-service
+```
+
+## Configuration
+
+To configure the microservice/Function, override the default values of these environment variables:
+
+| Environment variable | Description                                                                   | Required   | Default value |
+| ---------------------- | ----------------------------------------------------------------------------- | ------ | ------------- |
+| **APP_PORT**       | Specifies the port of running service. Function doesn't use this variable. | NO | `8080`           |
+| **APP_REDIS_PREFIX**       | Specifies the prefix for all related to Redis environment variables. See the variables below. | NO | `REDIS_`           |
+| **{APP_REDIS_PREFIX}HOST**       | Specifies the host of Redis instance.                       | NO | `nil`            |
+| **{APP_REDIS_PREFIX}PORT**       | Specifies the port of Redis instance.                       | NO | `nil`            |
+| **{APP_REDIS_PREFIX}REDIS_PASSWORD**       | Specifies the password of Redis instance to authorization.                       | NO | `nil`            |
+
+Example:
+
+```bash
+export APP_REDIS_PREFIX="R_"
+export R_HOST="abc.com"
+export R_PORT="8080"
+export R_REDIS_PASSWORD="xyz"
+```
+
+For communicate the microservice/Function with the Redis instance, the **{APP_REDIS_PREFIX}HOST**, **{APP_REDIS_PREFIX}PORT**, **{APP_REDIS_PREFIX}REDIS_PASSWORD** environments must be given. 
```

Comment:
```suggestion
> **NOTE:** To allow the microservice and the Function to communicate with the Redis instance, you must provide the **{APP_REDIS_PREFIX}HOST**, **{APP_REDIS_PREFIX}PORT**, **{APP_REDIS_PREFIX}REDIS_PASSWORD** environments. Otherwise, the microservice and the Function will always use in-memory storage.
```